### DISCUSSION TITLE
UNRESOLVED: how to fix things that are not working by myself?

### DISCUSSION THREAD

**Discussion by dasmou** [InnovativeOnlineIndustries/Industrial-Foregoing#1352] [Q&A]
I opened this bug quite a long time ago and still did get no answer on how to edit the file:

https://github.com/InnovativeOnlineIndustries/Industrial-Foregoing/issues/1105

I even found the reason of why it is buggy but unfortunately i cannot create a working java file with the fix in it. so PLEASE help me out.
mod devs say "it is not supported by us anymore" instead of reading what i actually said. 

thanks a lot!

**Comment by dasmou** [comment]
bump

**Comment by Buuz135** [comment]
1.12 is dead, I won't give any support for it

**Comment by dasmou** [comment]
im not asking you for support, i am asking you how i de- and re-compile the java file so it works in minecraft.

**Comment by dasmou** [comment]
any updates?
